### DISCUSSION TITLE
S-132 and the VGA connector.

### DISCUSSION THREAD

**Discussion by nbreeden2** [thehighnibble/thehighnibble.github.io#112] [S-132 add-on for IMSAI & Cromemco replicas]
On my S-132 the VGA connection points down, I believe they all do. With the connector pointing down it make it difficult to attach a VGA cable as it required a 90-degree bend in the cable to exit from the back of the case. I used an inexpensive VGA cable. At some point the red stopped working. At first, I thought this was an issue with the S-132, did a LOT of troubleshooting directly on the S-132, I even replaced the ESP32. In the end VGA cable had failed. I picked up a right-angle adapter on Amazon that redirects the VGA connection out the back and is short enough the IMSAI still sits correctly.
https://www.amazon.com/gp/product/B0851C21ZD/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1
And no, this isn't a sponsored link. :)
-Neil

**Comment by donfroula** [comment]
Nice idea. I had a thinner VGA cable in my box of display cables, but it still needs to bend more than I like.  Without the screw fastening the S132 in place, I was concerned about the board connector pulling free from the main board because of the stress caused by the bend in the cable. 

I dread removing the frame and back cover just to drill one hole with a step drill bit to secure the S132 with the supplied screw. The curve in the main board tends to push the screws out of holes, threatening the spacer alignment. 

Don
